modules/services/localization/src/main/res/values/strings.xml

```diff
@@ -350,6 +353,7 @@
     <string name="player_actions_hidden_for_custom">Hidden for custom episodes</string>
     <string name="player_actions_show_as_delete_for_custom">Shown as delete for custom episodes</string>
     <string name="player_sleep_time_fired">Sleep timer stopped at the end of your episode.\nNight night!</string>
+    <string name="player_sleep_time_fired_end_of_chapter">Sleep timer stopped at the end of your chapter.</string>
```

Comment:
Hey @david-gonzalez-a8c I am displaying this `Sleep timer stopped at the end of your chapter` message in the toast when the sleep timer has stopped for end of chapters. What do you think? Do you have another suggestion?

Reply:
cc: @leandroalonso

Reply:
We are going to keep this message. I confirme with David on slack